lost/src/main/java/com/mapzen/android/lost/internal/LocationSettingsResultRequest.java

```diff
@@ -40,8 +43,10 @@
 
   Future<LocationSettingsResult> future;
 
-  public LocationSettingsResultRequest(BluetoothAdapter btAdapter, PackageManager pm,
-      LocationManager lm, PendingIntentGenerator generator, LocationSettingsRequest request) {
+  public LocationSettingsResultRequest(Context context, BluetoothAdapter btAdapter,
```

Comment:
To avoid adding more constructor params we should use the `Context` to get a reference the `PackageManager` rather than passing both.